pcap-mode.el

```diff
@@ -72,12 +72,22 @@
 ;;;###autoload
 (defcustom pcap-mode-tshark-filter ""
   "Filter to apply to tshark invocations."
-  :type 'string)
+  :type 'string
```

Comment:
Please put this change, and the similar one below (for pcap-mode-tshark-single-packet-filter) in a separate commit.